src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: $product-card-width;
```

Comment:
Ensure that the variable `$product-card-width` is defined in your variables file, otherwise it will cause a compilation error.